src/rdkafka_telemetry.c

```diff
@@ -271,6 +280,19 @@ void rd_kafka_handle_push_telemetry(rd_kafka_t *rk, rd_kafka_resp_err_t err) {
                 rd_kafka_dbg(rk, TELEMETRY, "PUSHERR",
                              "PushTelemetryRequest failed: %s",
                              rd_kafka_err2str(err));
+                // Non-retriable errors
+                // TODO: What to do for TELEMETRY_TOO_LARGE?
```

Comment:
I guess we can just add a comment. Or at max a log (even though we will have logged a warning already), based on the discussion with the Java clients team, we need to do nothing special.

Reply:
Removed the comment